packages/components/markdown/src/remark-ui-component.ts

```diff
@@ -1,103 +1,219 @@
-import type { Dict } from "@yamada-ui/utils"
+import type { Status as AlertStatus } from "@yamada-ui/alert/src/alert"
```

Comment:
It would be better to export `AlertStatus` in `alert/index.ts`.
You can then import the following.
```ts
import type { AlertStatus } from "@yamada-ui/alert"
```